### PR TITLE
Cluster Local Gateway for Istio 1.1.6 - for KFServing

### PR DESCRIPTION
**Pull request by krishnadurai**
**Which issue is resolved by this Pull Request:**
Parially addresses kubeflow/kfserving#652

**Description of your changes:**

**Checklist:**
- [x] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate-changed-only`
    3. `make test`

/cc @yuzisun @animeshsingh 

/hold
For testing in a Kubeflow cluster

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/876)

### CODE REVIEW

**Comment by animeshsingh** [comment]
@krishnadurai 

@Tomcli tested in kubeflow cluster

File: istio/cluster-local-gateway/base/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,158 @@
+apiVersion: extensions/v1beta1
```
Comment (Tomcli) [inline istio/cluster-local-gateway/base/deployment.yaml]: Maybe Change it to v1 to support on Kubernetes 1.16?

Comment (krishnadurai) [reply istio/cluster-local-gateway/base/deployment.yaml]: Sure, I'll make this change.

Comment (Tomcli) [reply istio/cluster-local-gateway/base/deployment.yaml]: For apps/v1, `spec.selector` is now required under the deployment spec. Can you also update it according to this [blog](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/)? Thanks.

**Pull request merged**